src/Forms/GridField/GridFieldDataColumns.php

```diff
@@ -31,6 +29,15 @@ class GridFieldDataColumns extends AbstractGridFieldComponent implements GridFie
      */
     protected $displayFields = [];
 
+    private bool $displayStatusFlags = true;
+
+    private array $columnsForStatusFlag = [
+        'Title',
+        'Name',
+    ];
```

Comment:
These were the defaults in `VersionedGridFieldState` (see https://github.com/silverstripe/silverstripe-versioned/pull/479/files#diff-c3dbb0835d959d49ae741afb170b84162d26849c69dce38c55530e9338980b46)